fastlane/Fastfile

```diff
@@ -549,17 +548,50 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each do |source, destination|
+      source_dir = File.join(Dir.pwd, "metadata", source)
+      Dir.children(source_dir).each { |file| 
+        source_file = File.join(source_dir, file)
+        destination_file = File.join(Dir.pwd, "metadata", destination, file)
+        FileUtils.mv(source_file, destination_file)
+      }
+      FileUtils.remove_dir(source_dir)
+    end
+
+    # Copy source files into `en-US`
+    enUS_path = File.join(Dir.pwd, 'metadata', 'en-US')
+
+    FileUtils.mkdir_p(enUS_path)
 
-      support_path = Dir.pwd + "/metadata/#{locale}/support_url.txt"
-      write_file(support_path, "https://docs.woocommerce.com/")
+    [
+      "promo_screenshot_1.txt",
+      "promo_screenshot_2.txt",
+      "promo_screenshot_3.txt",
+      "promo_screenshot_4.txt",
+      "promo_screenshot_5.txt",
+    ].each { |filename|
+      source = File.join(Dir.pwd, 'appstoreres', 'metadata', 'source', filename)
+      destination = File.join(enUS_path, filename)
+      FileUtils.cp(source, destination)
     }
```

Comment:
Ditto about `do…end`

Reply:
Ah yep – fixed in 41e1543f718e8f25c1f8291538c48cdcdaa48644. 

I wonder if a linter could catch this for us? 😁